sentry/src/main/scala/com/avast/sst/sentry/SentryConfig.scala

```diff
@@ -1,10 +1,10 @@
 package com.avast.sst.sentry
 
 final case class SentryConfig(
-    dsn: String,
+    dsn: Option[String] = None,
```

Comment:
Why is this optional now? I don't think it can work without `DSN` so that's why it was mandatory.

Reply:
This is allowing the application to rely on the `SENTRY_DSN` environment variable instead of an explicit setting.
The downside of this is that since a programmer is not forced to provide the setting, she _might_ forget to set such a variable, but I was thinking that would happen rarely, if ever.
But no strong feelings either way. What do you think?

Reply:
I would like to keep the original code. Multiple points:
- It's quite strange to have a config with everything optional.
- `SENTRY_DSN` is a fallback behavior of the library and you can as well set it in HOCON.
- This library is supposed to unify the way things are configured so it does not make sense to configure one of the components using env variable only.